voter/models.py

```diff
@@ -241,13 +257,17 @@ def data_from_row(cls, row):
         row['registr_dt'] = str(row['registr_dt'])
         return row, {}
 
-    def build_current(self):
+    def build_version(self, index):
         changelog = self.changelog.all()
         data = {}
-        for change in changelog:
+        nindex = len(changelog) - index
+        for change in list(changelog)[:nindex]:
```

Comment:
is this the same as ```[:-index]```?

Reply:
I actually thought so,  looking back, but it looks like that breaks the tests so no.